tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -3081,10 +3077,9 @@ declare namespace ts {
             readonly containingProjects: Project[];
             private formatSettings;
             private preferences;
-            private textStorage;
-            constructor(host: ServerHost, fileName: NormalizedPath, scriptKind: ScriptKind, hasMixedContent: boolean, path: Path, initialVersion?: ScriptInfoVersion);
+            constructor(host: ServerHost, fileName: NormalizedPath, scriptKind: ScriptKind, hasMixedContent: boolean, path: Path, initialVersion?: number);
```

Comment:
Given this breaking change, is anyone depending on this taht we should worry about? Is there a way to write this code as to not break existing users, i.e. keep the old thing around, deprecate, and pull a version out of it?

Reply:
I have looked up but didn't find any usage for this which is why i just modified this without adding overload.
You think overload thats marked deprecated is better even then?

Reply:
Yeah, now that I'm looking, I've only been able to find 3 examples of this being used (a subset of https://sourcegraph.com/search?q=context:global+/new+.*%5CbScriptInfo%5C%28/+lang:typescript+-file:harness+-file:editorServices+-file:languageServiceHost&patternType=standard&sm=1&groupBy=repo are the right thing), and those all a decade old and unmaintained. So, probably safe.